src/support_code_library_builder/world.ts

```diff
@@ -6,7 +6,14 @@ export interface IWorldOptions {
   parameters: any
 }
 
-export default class World {
+export interface IWorld {
+  readonly attach: ICreateAttachment
+  readonly log: ICreateLog
+  readonly parameters: any
```

Comment:
Any possibility to avoid the `any` type here?
It is pluralize. Is it an array? Or something else?

Reply:
This will be deserialized from `---world-parameters` which could be any JSON object. We could try and constrain the type accordingly but...[it's a bit of a rabbit hole](https://github.com/microsoft/TypeScript/issues/1897).

Reply:
Thanks for the clarification